src/cameras/capture/opencv_camera/opencv_camera.py

```diff
@@ -38,6 +56,23 @@ def __init__(self, config: WebcamConfig):
         self._name = f"Camera {self._config.webcam_id}"
         self._opencv_video_capture_object: cv2.VideoCapture = None
         self._running_thread: FrameThread = None
+        self._camera_calibration_data = None
+
+    @property
+    def camera_calibration_data(self):
+        if self._camera_calibration_data is not None:
+            return self._camera_calibration_data
+
+        if self._camera_calibration_data is None and self.image_width is not None:
+            self._camera_calibration_data = CameraCalibrationData(self.image_width, self.image_height)
+            return self._camera_calibration_data
+
+        logger.warning(
+            'Can\'t request a cameras calibration info until the video capture is initialized and the image_width and image_height are known')
+
+    @camera_calibration_data.setter
+    def camera_calibration_data(self, camera_calibration_data: CameraCalibrationData):
```

Comment:
this indicates that we would be setting calibration data from within the OpenCVCamera class - i think that breaks our current definition of what OCVCam does.

Reply:
agreed - this'll get sorted out when I refactor to align with that pipeline diagram I sent you earlier. I'll remove this stuff for now and make sure it leaves things functional

Reply:
ah okay understood